packages/mui-codemod/src/deprecations/pagination-item-props/test-cases/theme.actual.js

```diff
@@ -0,0 +1,25 @@
+fn({
+  MuiPaginationItem: {
+    defaultProps: {
+      slots: {
```

Comment:
Looks like this should've been `components`, same as the one below.

Reply:
fixed here https://github.com/mui/material-ui/pull/41777/commits/5329aafb26b5a49a6e6d46ddaf2792ffe7d104cf